modules/social_features/social_post/social_post.module

```diff
@@ -78,16 +78,18 @@ function social_post_form_comment_post_comment_form_alter(&$form, FormStateInter
   // Set the action of the form to the current uri.
   // This needs to be done, because we cannot override $form['#action']
   // without breaking functionality.
-  $uri = Url::fromRoute('<current>')->toString();
-  // Store in a hidden field.
-  $form['redirect_after_save'] = [
-    '#type' => 'hidden',
-    '#title' => t('Redir'),
-    '#default_value' => $uri,
-  ];
-
-  // Submit function to retrieve the action uri and redirect to it.
-  $form['actions']['submit']['#submit'][] = '_social_post_comment_post_comment_form_submit';
+  if (!\Drupal::moduleHandler()->moduleExists('ajax_comments')) {
```

Comment:
Lets add a comment here on why this is and move the existing comment of what is happening into the if statement :)